### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by nmartinelli23**
## Summary
- Implemented the footer layout.
- I don't know why i couldn't make the elements of this section align respecting the feature section.
## Screenshots

captura de pantalla de 2016-04-21 16-12-04
## Trello Card

https://trello.com/c/9RBYBWMm

### CODE REVIEW

File: ejemplo.html [hunk 1]
```diff
@@ -35,4 +35,14 @@ <h1 class="feature-title">No internet needed</h1>
   
 </body>
 
+<footer class="shadow-add">
+	
+  <form class="form-container">
+    <h1 class="footer-title">Get info on new updates</h1>
+    <input class="footer-box" type = "text" id = "new-email" />
```
Comment (SKOLZ) [inline ejemplo.html]: don't leave spaces around = in html

```
type="text" id="new-email"
```

File: styles.css [hunk 2]
```diff
@@ -65,3 +70,49 @@
...
+  flex-direction: column;
+ }
+ 
+.footer-title {
+  font-size: 16px;
+  color: #EEE;
+ }
+ 
+.footer-box[type=text] {
+  padding:5px 25px;
```
Comment (SKOLZ) [inline styles.css]: space after :

File: styles.css [hunk 3]
```diff
@@ -2,18 +2,21 @@
...
-
+ 
 * {
   font-family: "myriadPro", Sans-Serif;
+  outline: 0;
  }
 
-.main-container {
-  border-top: 5px solid #FFA500;
+.main-container{
```
Comment (SKOLZ) [inline styles.css]: space before curly brace

File: styles.css [hunk 4]
```diff
@@ -65,3 +70,49 @@
...
+  color: #000;
+  width: 250px;
+  height: 30px;
+  margin-top: 10px;  
+  margin-bottom: 20px;  
+  border-radius: 10px;
+  border: none;
+ }
+
+input#footer-botton {
```
Comment (SKOLZ) [inline styles.css]: don't use ids to target elements in css, also don't use HTML elements for that too,  just add a class

**Comment by SKOLZ** [comment]
👍

**Comment by nmartinelli23** [comment]
I correct all the coments made. This is a new screeshot:

captura de pantalla de 2016-04-25 11-12-37

**Comment by SKOLZ** [comment]
👍

**Pull request merged**